src/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -64,11 +67,14 @@ See the LICENSE file in the project root for more information.
 
     <SharedLibrary Condition="'$(OS)' == 'Windows_NT'">$(FrameworkLibPath)\Framework$(LibFileExt)</SharedLibrary>
     <SharedLibrary Condition="'$(OS)' != 'Windows_NT'">$(FrameworkLibPath)\libframework$(LibFileExt)</SharedLibrary>
+    <DynamicBuildPropertyDependencies Condition="'$(CalledViaPackage)' == 'true'">SetupProperties</DynamicBuildPropertyDependencies>
```

Comment:
It may be useful to have `Ilc` prefix e.g. `IlcDynamicBuildPropertyDependencies` for all these generically looking properties. It would make it obvious that the property is specific to Ilc, not a msbuild built-in.